community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/commands/expressions/StringFunctionsTest.scala

```diff
@@ -119,6 +121,7 @@ class StringFunctionsTest extends CypherFunSuite {
     ltrim("  hello") should equal(stringValue("hello"))
     ltrim("  hello  ") should equal(stringValue("hello  "))
     ltrim(null) should equal(expectedNull)
+    ltrim("\u2009㺂࿝鋦毠\u2009") should equal(stringValue("㺂࿝鋦毠\u2009"))//Contains `thin space`
```

Comment:
Could we do some sort of exhaustive testing for this? I.e. generate strings padded with all possible whitespaces?

Reply:
I have added scalacheck test in `TextValueSpecification` thought it was better to do the exhaustive testing on the underlying values instead of on a specific runtime implementation